addons/web/static/src/boot.js

```diff
@@ -440,4 +430,60 @@
         // visit each root to find cycles
         return visitJobs(jobs.map((j) => j.name));
     }
+
+    function displayModuleErrors({ failed, missing, unloaded, cycle }) {
+        const list = (heading, arr) => {
+            const frag = document.createDocumentFragment();
+            if (!arr || !arr.length) {
+                return frag;
+            }
+            frag.textContent = heading;
+            const ul = document.createElement("ul");
+            for (const el of arr) {
+                const li = document.createElement("li");
+                li.textContent = el;
+                ul.append(li);
+            }
+            frag.appendChild(ul);
+            return frag;
+        };
+        if ([failed, missing, unloaded].some((arr) => arr.length) || cycle) {
+            // Empty body
+            while (document.body.childNodes.length) {
+                document.body.childNodes[0].remove();
+            }
```

Comment:
FWIW you can set textContent to clear the body (it should be even faster than doing so via innerHTML, since the browser doesn't need to run the HTML parser if it failed to fast-path the operation).